There is still the following error visible in the NixOS tests.

```
vm-test-run-paperless> subtest: Documents are consumed
vm-test-run-paperless> machine: waiting for success: (($(curl -u admin:admin -fs localhost:28981/api/documents/ | jq .count) == 2))
vm-test-run-paperless> machine # [   29.821756] paperless-ngx[1071]: [2022-09-04 12:21:28,810] [INFO] [paperless.consumer] Consuming doc.png
vm-test-run-paperless> machine # [   30.666513] paperless-ngx[1071]: [2022-09-04 12:21:29,655] [WARNING] [paperless.parsing.tesseract] Error while getting DPI from image /var/lib/paperless'
vm-test-run-paperless> machine # [   33.225031] paperless-ngx[1071]: [2022-09-04 12:21:32,213] [WARNING] [ocrmypdf._exec.tesseract] [tesseract] Warning: Invalid resolution 48 dpi. Using 70.
vm-test-run-paperless> machine # [   33.226829] paperless-ngx[1071]: [2022-09-04 12:21:32,213] [WARNING] [ocrmypdf._exec.tesseract] [tesseract] Warning. Invalid resolution 48 dpi. Using 70.
vm-test-run-paperless> machine # [   33.229178] paperless-ngx[1071]: [2022-09-04 12:21:32,213] [ERROR] [ocrmypdf._exec.tesseract] [tesseract] Error during processing.
vm-test-run-paperless> machine # [   34.208424] paperless-ngx[1071]: [2022-09-04 12:21:33,197] [WARNING] [ocrmypdf._exec.tesseract] [tesseract] Warning: Invalid resolution 48 dpi. Using 70.
vm-test-run-paperless> machine # [   34.633477] paperless-ngx[1153]: convert: FailedToExecuteCommand `'gs' -sstdout=%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=5000000.
vm-test-run-paperless> machine # [   34.638537] paperless-ngx[1153]: convert: no images defined `/tmp/paperless/paperless-aid36n32/convert.webp' @ error/convert.c/ConvertImageCommand/3342.
vm-test-run-paperless> machine # [   34.656765] paperless-ngx[1071]: [2022-09-04 12:21:33,645] [WARNING] [paperless.parsing] Thumbnail generation with ImageMagick failed, falling back to g!
vm-test-run-paperless> machine # [   35.650601] paperless-ngx[1071]: [2022-09-04 12:21:34,639] [INFO] [paperless.consumer] Document 2005-10-16 doc consumption finished
vm-test-run-paperless> machine # [   35.653660] paperless-ngx[1071]: 12:21:34 [Q] INFO Process-1:1 stopped doing work
vm-test-run-paperless> machine # [   35.671751] paperless-ngx[1073]: 12:21:34 [Q] INFO Processed [doc.png]
vm-test-run-paperless> machine # [   38.267439] paperless-ngx[1070]: 12:21:37 [Q] INFO recycled worker Process-1:1
vm-test-run-paperless> machine # [   38.270700] paperless-ngx[1180]: 12:21:37 [Q] INFO Process-1:4 ready for work at 1180
vm-test-run-paperless> machine # [   38.271560] paperless-ngx[1180]: 12:21:37 [Q] INFO Process-1:4 processing [webdoc.png]
vm-test-run-paperless> machine # [   38.890814] paperless-ngx[1180]: [2022-09-04 12:21:37,879] [INFO] [paperless.consumer] Consuming webdoc.png
vm-test-run-paperless> machine # [   39.224800] paperless-ngx[1180]: [2022-09-04 12:21:38,213] [WARNING] [paperless.parsing.tesseract] Error while getting DPI from image /tmp/paperless/pap'
vm-test-run-paperless> machine # [   40.001992] paperless-ngx[1180]: [2022-09-04 12:21:38,990] [WARNING] [ocrmypdf._exec.tesseract] [tesseract] Warning: Invalid resolution 48 dpi. Using 70.
vm-test-run-paperless> machine # [   40.002783] paperless-ngx[1180]: [2022-09-04 12:21:38,990] [WARNING] [ocrmypdf._exec.tesseract] [tesseract] Warning. Invalid resolution 48 dpi. Using 70.
vm-test-run-paperless> machine # [   40.004080] paperless-ngx[1180]: [2022-09-04 12:21:38,990] [ERROR] [ocrmypdf._exec.tesseract] [tesseract] Error during processing.
vm-test-run-paperless> machine # [   40.756717] paperless-ngx[1180]: [2022-09-04 12:21:39,745] [WARNING] [ocrmypdf._exec.tesseract] [tesseract] Warning: Invalid resolution 48 dpi. Using 70.
vm-test-run-paperless> machine # [   41.041213] paperless-ngx[1224]: convert: FailedToExecuteCommand `'gs' -sstdout=%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=5000000.
vm-test-run-paperless> machine # [   41.044443] paperless-ngx[1224]: convert: no images defined `/tmp/paperless/paperless-h13cqq46/convert.webp' @ error/convert.c/ConvertImageCommand/3342.
vm-test-run-paperless> machine # [   41.066114] paperless-ngx[1180]: [2022-09-04 12:21:40,054] [WARNING] [paperless.parsing] Thumbnail generation with ImageMagick failed, falling back to g!
vm-test-run-paperless> machine # [   41.985678] paperless-ngx[1180]: [2022-09-04 12:21:40,974] [INFO] [paperless.consumer] Document 2005-10-16 webdoc consumption finished
vm-test-run-paperless> machine # [   41.988824] paperless-ngx[1180]: 12:21:40 [Q] INFO Process-1:4 stopped doing work
vm-test-run-paperless> machine # [   41.998447] paperless-ngx[1073]: 12:21:40 [Q] INFO Processed [webdoc.png]
vm-test-run-paperless> (finished: waiting for success: (($(curl -u admin:admin -fs localhost:28981/api/documents/ | jq .count) == 2)), in 14.13 seconds)

--- a/nixos/modules/services/misc/paperless.nix
+++ b/nixos/modules/services/misc/paperless.nix
@@ -176,11 +176,10 @@ in
         See [the documentation](https://paperless-ngx.readthedocs.io/en/latest/configuration.html)
         for available options.
       '';
-      example = literalExpression ''
-        {
-          PAPERLESS_OCR_LANGUAGE = "deu+eng";
-        }
-      '';
+      example = {
+        PAPERLESS_OCR_LANGUAGE = "deu+eng";
+        PAPERLESS_DBHOST = "/run/postgresql";
+      };
     };
 
     user = mkOption {

--- a/nixos/modules/services/misc/paperless.nix
+++ b/nixos/modules/services/misc/paperless.nix
@@ -287,8 +287,8 @@ in
 
         AmbientCapabilities = "CAP_NET_BIND_SERVICE";
         CapabilityBoundingSet = "CAP_NET_BIND_SERVICE";
-        # gunicorn needs setuid
-        SystemCallFilter = defaultServiceConfig.SystemCallFilter ++ [ "@setuid" ];
+        # gunicorn needs setuid, liblapack needs mbind
+        SystemCallFilter = defaultServiceConfig.SystemCallFilter ++ [ "@setuid mbind" ];
         # Needs to serve web page
         PrivateNetwork = false;
       };

--- a/pkgs/applications/office/paperless-ngx/default.nix
+++ b/pkgs/applications/office/paperless-ngx/default.nix
@@ -16,7 +16,7 @@
 
 let
   # Use specific package versions required by paperless-ngx
-  py = python3.override {
+  python = python3.override {
     packageOverrides = self: super: {
       django = super.django_4;
 
@@ -51,7 +51,7 @@ let
 
   path = lib.makeBinPath [ ghostscript imagemagick jbig2enc optipng pngquant qpdf tesseract4 unpaper ];
 in
-py.pkgs.pythonPackages.buildPythonApplication rec {
+python.pkgs.pythonPackages.buildPythonApplication rec {
   pname = "paperless-ngx";
   version = "1.8.0";
 
@@ -63,7 +63,7 @@ py.pkgs.pythonPackages.buildPythonApplication rec {
 
   format = "other";
 
-  propagatedBuildInputs = with py.pkgs.pythonPackages; [
+  propagatedBuildInputs = with python.pkgs.pythonPackages; [
     aioredis
     arrow
     asgiref
@@ -169,7 +169,7 @@ py.pkgs.pythonPackages.buildPythonApplication rec {
   # Compile manually because `pythonRecompileBytecodeHook` only works for
   # files in `python.sitePackages`
   postBuild = ''
-    ${py.interpreter} -OO -m compileall src
+    ${python.interpreter} -OO -m compileall src
   '';
 
   installPhase = ''
@@ -181,7 +181,7 @@ py.pkgs.pythonPackages.buildPythonApplication rec {
       --prefix PATH : "${path}"
   '';
 
-  checkInputs = with py.pkgs.pythonPackages; [
+  checkInputs = with python.pkgs.pythonPackages; [
     pytest-django
     pytest-env
     pytest-sugar
@@ -207,6 +207,7 @@ py.pkgs.pythonPackages.buildPythonApplication rec {
   '';
 
   passthru = {
+    inherit python;
     # PYTHONPATH of all dependencies used by the package
     pythonPath = python3.pkgs.makePythonPath propagatedBuildInputs;
     inherit path;

